java/compaction/compaction-core/src/test/java/sleeper/compaction/job/CompactionJobStatusInPeriodTest.java

```diff
@@ -151,9 +152,10 @@ public void shouldIncludePeriodWithFinishedUpdateTime() {
         Instant beforeTime2 = Instant.parse("2022-09-23T11:43:01.000Z");
         Instant startTime = Instant.parse("2022-09-23T11:44:00.000Z");
         Instant endTime = Instant.parse("2022-09-23T11:44:01.000Z");
-        Instant afterTime = Instant.parse("2022-09-23T11:44:02.000Z");
+        Instant afterTime1 = Instant.parse("2022-09-23T11:44:02.000Z");
+        Instant afterTime2 = Instant.parse("2022-09-23T11:44:05.000Z");
```

Comment:
It might be worth putting the finished time before the query period. If we're consistent about this between the tests, we can be sure it's not the finished time that matters, just the last time.